### PR TITLE
Using ubi8 for iib messaging container image

### PR DESCRIPTION
**Pull request by lipoja**
This will help us to reduce security warnings on queay.io

[CLOUDDST-12355]

### CODE REVIEW

**Comment by JAVGan** [comment]
I believe you can squash the commits, right?

I see the 013375caf54b7cfd29f4b54e51572d363ce5b6bb to run `apt update` which is later on replaced by `dnf`

**Comment by lipoja** [comment]
Oh noeas, my bad. Thanks @JAVGan

File: docker/message_broker/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Based from https://github.com/rmohr/docker-activemq
-FROM openjdk:8-jre
+FROM registry.access.redhat.com/ubi8/ubi:latest
 
 ENV ACTIVEMQ_VERSION 5.15.12
 ENV ACTIVEMQ_HOME /opt/activemq
 
+# keeping container image updated (security updates)
+RUN  dnf install java-1.8.0-openjdk maven -y && dnf update -y
```
Comment (chandwanitulsi) [inline docker/message_broker/Dockerfile:8]: Hey Jan,
Do you know if we specifically need Java 1.8?, I believe we have the latest Java 11 and Java 17 available as well.
I am thinking that maybe using latest java will give less security issues

Comment (lipoja) [reply docker/message_broker/Dockerfile:8]: That is good point! I will check that.

**Pull request merged**